syntax = "proto3";

package endpoints;

import "models/registered_function.proto";

message CheckFunctionRequest {
  optional models.RegisteredFunction function_to_check = 1;
}

// A successful response indicates the function is registered.
// If the function is not regsitered, a NOT_FOUND status will be thrown.
message CheckFunctionResponse {
}
